acceptance/tests/restart_host_run_puppet.rb

```diff
@@ -22,8 +22,14 @@
   step "restart each agent" do
     applicable_agents.each do |agent|
       agent.reboot
+      # BKR-812
+      sleep 1 if !agent.up?
```

Comment:
Why not sleep for longer, or test agent.up? multiple times? I'm not sure it's possible to test that 1 second sleep will make a difference for the Ubuntu 14.04 problem; and there's a risk that at some point any other host might need longer